CMakeLists.txt

```diff
@@ -180,6 +180,9 @@ install(FILES icon.png package-info.json QuickStart.md
 install(FILES ${flucoma-core_SOURCE_DIR}/distribution.lic 
         DESTINATION ${MAX_PACKAGE_ROOT} 
         RENAME LICENSE.md)
+install(DIRECTORY local_docs/
+        DESTINATION ${MAX_PACKAGE_ROOT}/docs)
```

Comment:
can you wrap the folder names in quotes in case there's spaces in the parent path. Also, probably more robust to specify an absolute path for `local_docs` in terms of the source dir: `"${CMAKE_CURRENT_SOURCE_DIR}/local_docs"`

Reply:
Agree

Reply:
Ah, sorry. Can you en-quote `${MAX_PACKAGE_ROOT}/docs` as well?